src/Symfony/Component/Security/Http/EntryPoint/BasicAuthenticationEntryPoint.php

```diff
@@ -14,11 +14,16 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Http\Authenticator\HttpBasicAuthenticator;
+
+trigger_deprecation('symfony/security-http', '5.4', 'The "%s" class is deprecated, use the new security system with "%s" instead.', BasicAuthenticationEntryPoint::class, HttpBasicAuthenticator::class);
 
 /**
  * BasicAuthenticationEntryPoint starts an HTTP Basic authentication.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated since 5.4
```

Comment:
```suggestion
 * @deprecated since Symfony 5.4
```
(same everywhere)

Reply:
see #42534 that fixes the current inconsistencies with "the rule".

Reply:
fabbot reports 3 such mistakes now :)